packages/blade/src/components/Skeleton/_decisions/decisions.md

```diff
@@ -0,0 +1,327 @@
+# Skeleton <!-- omit in toc -->
+Skeleton Loader is a static / animated placeholder for the information that is still loading. It mimic the structure and look of the entire view. 
+
+The skeleton blocks can have 3 types of shape
```

Comment:
do we need to mention this explicitly? we have borderadius to tackle this already which is also implied? right this is leads to a confusion that we'll have a variant as a prop which is not the case

Reply:
Makes sense, removed.